doc/Type/List.pod6

```diff
@@ -277,7 +277,7 @@ Defined as:
     method join(List:D: $separator = "")
 
 Treats the elements of the list as strings by calling
-L<C<.Str>|/type/Any#routine_Str> on each of them, interleaves them with
```

Comment:
What about changing it to `/type/Mu#method_Str`? It probably makes more sense. It's also pretty much the correct method, taking into account any object will subclass `Mu`